Project 2 (13.12)/script.js

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <script>
+      ZADANIE 1
+        function OdwrotnoscHasla() {
+          let string = "piesek";
+          string = [...string].reverse().join("");
+          return string;
+        }
+        console.log(OdwrotnoscHasla());
+
+        ZADANIE 2
+
+        function PODMIANKA() {
+          const string = "Joanna Czarnecka";
+          const [Imie, Nazwisko] = string.split(" ");
+          const PierwszaLiterkaImie = Imie[0];
+          const PierwszaLiterkaNazwisko = Nazwisko[0];
+          const ZamienioneLiteryImie = PierwszaLiterkaNazwisko + Imie.slice(1);
+          const ZamienioneLiteryNaziwsko =
+            PierwszaLiterkaImie + Nazwisko.slice(1);
+          const Zamienione =
+            ZamienioneLiteryImie + " " + ZamienioneLiteryNaziwsko;
+          return Zamienione;
+        }
+        console.log(PODMIANKA());
```

Comment:
tutaj to samo. const string w środku powoduje, że Twoja funkcja robi tylko jedną rzecz z jednym stringiem.